annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -782,11 +793,6 @@ private String getTumorSeqAllele(MutationRecord record) {
         }
     }
 
-    @Override
-    public boolean isHgvspNullClassifications(String variantClassification) {
-        return hgvspNullClassifications.contains(variantClassification);
-    }
-
     private static List<String> initNullClassifications() {
```

Comment:
As it is an interface method, I moved it next to the others. Where would you suggest it be? It's not used in this class, it's basically a way to interact with the list of null classifications described here.